### PR TITLE
Rewrite descriptions of corpses

### PR DESCRIPTION
**Pull request by aoae23**
#### Summary
```SUMMARY: Interface "Rewrite some corpse descriptions"```

#### Purpose of change
Rewrote descriptions of some of the newly added corpses in #33795. The goal was to keep the horror intended by the author intact, while keeping the descriptions mostly free of "telling" rather than "showing" (and fixing a few grammatical errors).

#### Describe alternatives you've considered
Not rewriting them

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
I don't like this. I see nothing wrong in "telling"-style descriptions.

File: data/json/items/corpses.json [hunk 1]
```diff
@@ -136,7 +137,7 @@
     "id": "corpse_gunned",
     "copy-from": "corpse",
     "name": "corpse",
-    "description": "One might say that face of this woman is devoid of flaws. That could be true if there wasn't this big ugly hole right in the center of her forehead, and even bigger hole in the back of her head.",
+    "description": "A dead body of a woman, with a large gunshot wound in her forehead and a corresponding hole in the back of her head.",
```
Comment (tenmillimaster) [inline data/json/items/corpses.json]:
```suggestion
    "description": "The dead body of a person.  Their forehead bears a large bullet entrance wound.  An even larger exit wound is present on the back of their head.",
```

File: data/json/items/corpses.json [hunk 2]
```diff
@@ -158,7 +159,7 @@
     "id": "corpse_half_beheaded",
     "copy-from": "corpse",
     "name": "corpse",
-    "description": "The remains of lower jaw and cheeks are still visible on this half-decapitated body.  Looks like half of its head has been torn off after hit of a high-caliber bullet.  Or maybe it was bit off by some huge beast?",
+    "description": "A half-decapitated dead body. It is unclear what could have caused such a wound.",
```
Comment (tenmillimaster) [inline data/json/items/corpses.json]:
```suggestion
    "description": "A half-decapitated dead body.  It is unclear what could have caused such a wound.",
```

**Comment by tenmillimaster** [comment]
> I don't like this. I see nothing wrong in "telling"-style descriptions.

Descriptions that do not force viewpoint or tone are better and match the rest of CDDA.

File: data/json/items/corpses.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "symbol": "%",
     "color": "white",
     "name": "desiccated corpse",
-    "description": "A badly mangled and desiccated partial corpse.  It seems whatever thing killed him did so with a single swipe of a gigantic claw.",
+    "description": "A dead body, badly mangled and desiccated.  It seems whatever killed him did so with a gigantic claw.",
```
[the remark this answers is not in the document]
Comment (aoae23) [reply data/json/items/corpses.json]: Pyranik intended the gendered descriptions so I tried to preserve them for the most part.

[the remark this answers is not in the document]
Comment (tenmillimaster) [reply data/json/items/corpses.json]: I think genderless is probably better, unless they really need to be gendered (grandma with jewellery)

**Comment by Night-Pryanik** [comment]
> Descriptions that do not force viewpoint or tone are better and match the rest of CDDA.

Firstly, there are A LOT of descriptions that "force viewpoint or tone". 
Secondly, why are they better? Do you really think that compact, dry, official, criminalist-report-style "A dead body with a through gunshot wound of the head" description is better than an evocative, emotional one?

File: data/json/items/corpses.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (John-Candlebury) [reply data/json/items/corpses.json]: That corpse is canonically male. He is the previous Hub 01 Director.

**Comment by tenmillimaster** [comment]
> > Descriptions that do not force viewpoint or tone are better and match the rest of CDDA.
> 
> Firstly, there are A LOT of descriptions that "force viewpoint or tone".
> Secondly, why are they better? Do you really think that compact, dry, official, criminalist-report-style "A dead body with a through gunshot wound of the head" description is better than an evocative, emotional one?

Yes. That is what other devs have said before in other discussions, as they wish for emergent gameplay and roleplaying to occur.

**Comment by Night-Pryanik** [comment]
> Yes. That is what other devs have said before in other discussions, as they wish for emergent gameplay and roleplaying to occur.

Well, I haven't seen any such statements (though I could have missed them). I'll ask @kevingranade directly - is my descriptions ok, or is it better to rewrite them as @aoae23 and @tenmillimaster suggest?

File: data/json/items/corpses.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Soul5hadows) [reply data/json/items/corpses.json]: When a person's gender is unknown, he is referred to in masculine terms.

[the remark this answers is not in the document]
Comment (tenmillimaster) [reply data/json/items/corpses.json]: Ah, thanks candlebury.

**Comment by harald921** [comment]
The description is meant to describe what you are looking at, not what your character thinks of it - for what your character thinks is highly dependent on what you role play the character as, and the situation.

**Comment by Fris0uman** [comment]
For what it's worth (not much), I think the initial descriptions were fine an didn't really force any thought on the player, and that the new ones use less word but still manage to carry the same feelings.

**Comment by Night-Pryanik** [comment]
If I'd knew my descriptions ain't suitable, I'd never started that new human corpses PR at all.

**Comment by latogato** [comment]
It is a horror game, reading a sterile description which fail to trigger emotions is boring. Here is a similar kind of erotic story: two persons met and they slept together. THE END. You see, it sucks.

**Comment by tenmillimaster** [comment]
Read Cormac McCarthy's the road.

Not watch, read.

You'll understand how dry detachment can have its own sense of horror.

**Comment by Night-Pryanik** [comment]
Maybe McCarthy was able to portray horror by using dry descriptions, but in my opinion in case of this PR this goal wasn't achieved.

**Pull request merged**